RandomImage/app.js

```diff
@@ -0,0 +1,43 @@
+console.log("hi");
+
+
+
+// the type of the event 
+const eventType = 'click';
+
+// the target of the event 
+
+const myButton = document.querySelector('button#randomize');
+
+// the callback - what do you want to do as a response?
+const myCallback = function(){
+ const img = document.createElement("img");
+ const imges = ["https://i.picsum.photos/id/1002/4312/2868.jpg", 
```

Comment:
we want you to try using the API 
by creating a new variable called URL, so you can get random pictures from unsplash.

```
    let url = ("https://source.unsplash.com/1600x900");
    let image = document.querySelector(".img");
    image.setAttribute("src", url);
```

if you want to make it so the the picture changes with a different height and width you can create a variable bor both and use it in the URL
```
let width = document.querySelector("input.width");
let height = document.querySelector("input.height");
let url = ("https://source.unsplash.com/" + width + "x" + height);
let image = document.querySelector(".img");
image.setAttribute("src", url);


```